persistence/scaladsl/src/main/java/com/lightbend/lagom/internal/scaladsl/persistence/protobuf/msg/PersistenceMessages.java

```diff
@@ -0,0 +1,1780 @@
+/*
```

Comment:
It should be in version control, even if it's generated (at least that is how it's done for the javadsl).